### PR TITLE
refactor(vue-tutorial): move Meteor tutorial to vue3

### PR DESCRIPTION
**Pull request by vit0rr**
Hi!!
This PR is dedicated to refactoring every vue-tutorial to vue3. I'll also include a tutorial about prettier and eslint to vue get autosuggestion at IDE. Also, use Vue router, tailwindcss, and the newest version of MeteorJS.

And also using composition.
Issue ref.: https://github.com/meteor/vue-tutorial/issues/20

By the way, I'm learning vue during this process. So, if you note something that can be improved, please let me know.

#### Code
- [x] step01
- [x] step02
- [x] step03
- [x] step04
- [x] step05
- [x] step06
- [x] step07
- [x] step08
- [x] step09
- [x] step10
- [x] step11
- [x] step12

#### Documentation (markdown)
- [x] step01
- [x] step02
- [x] step03
- [x] step04
- [x] step05
- [x] step06
- [x] step07
- [x] step08
- [x] step09
- [x] step10
- [x] step11
- [x] step12
- [x] step13

https://github.com/meteor/vue-tutorial/assets/70543018/2ad25adc-40c0-46f6-ad91-875408520c7b

### CODE REVIEW

File: tutorial/simple-todos/01-creating-app.md [hunk 1]
```diff
@@ -32,99 +29,175 @@ meteor run
...
+## 1.3: Install prettier and eslint
+Vue do not have native support on vscode for example. To configure Vue on your favorite IDE you'll need to install and configure prettier and eslint.
+
+To install prettier and eslint, run:
+```
+meteor npm install --save-dev prettier @prettier/plugin-php
+```
+
+```
+meteor npm install --save-dev eslint eslint-plugin-vue eslint-config-prettier eslint-plugin-prettier
```
Comment (Grubba27) [inline tutorial/simple-todos/01-creating-app.md:45]: could this be one big command?
and plugin PHP is really necessary?

```suggestion

meteor npm install --save-dev eslint eslint-plugin-vue eslint-config-prettier eslint-plugin-prettier prettier @prettier/plugin-php

```

File: tutorial/simple-todos/01-creating-app.md [hunk 2]
```diff
@@ -9,13 +9,10 @@ Install the latest official Meteor release [following the steps in our docs](htt
...
 
-The easiest way to setup Meteor with Vue is by using the command `meteor create` with the option `--vue-2` and your project name:
-
-
-> Side note: Since meteorJS version 2.9 the default --vue will create a vue3 app, in order to follow this tutorial, use the vue2 command: ``--vue-2``
+The easiest way to setup Meteor with Vue is by using the command `meteor create` with the option `--vue` and your project name:
 
 ```
-meteor create --vue-2 simple-todos-vue --prototype
+meteor create --vue simple-todos-vue --prototype
```
Comment (Grubba27) [inline tutorial/simple-todos/01-creating-app.md:15]: later on you ask the user to add autopublish package, you should either pass the `--prototype` or ask the user to add the autopublish package

Comment (Grubba27) [reply tutorial/simple-todos/01-creating-app.md:15]: https://github.com/meteor/meteor/blob/cb6c97dc5f9897b61a5479d415a61ae2b61d1fac/tools/cli/commands.js#L1054-L1062

**Review by Grubba27 — CHANGES_REQUESTED** [review]
I would like to see those changes before this becomes a new tutorial. And recommend asking @henriquealbert and @fredmaiaarantes for review in this PR as well

**Review by henriquealbert — CHANGES_REQUESTED** [review]
I would remove the prototype part from the tutorial and also separate in 2 tutorials: vue2 and vue3

File: tutorial/simple-todos/01-creating-app.md (hunk 2 above)
Comment (Grubba27) [reply tutorial/simple-todos/01-creating-app.md:15]: I think we could change this
Maybe removing this `--prototype` and later on asking the user for the autopublish and other package that you have already mentioned

Comment (Grubba27) [reply tutorial/simple-todos/01-creating-app.md:15]: Asking the user to add `autopublish` and `insecure` sounds good

**Comment by vit0rr** [comment]
I think that we can close this PR since it is moved to https://github.com/meteor/vue3-tutorial/pull/2

**Pull request closed without merging**